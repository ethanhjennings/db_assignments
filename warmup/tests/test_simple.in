cat
CAT
whale
dog
lizard
.quit
