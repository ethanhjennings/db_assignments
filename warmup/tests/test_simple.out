Loading 'simple_dict.json'...
Loaded 3 words.
Enter a word to lookup:
------------------------------
CAT:
A four legged mammal.
Not to be confused with "dog".
------------------------------
Enter a word to lookup:
------------------------------
CAT:
A four legged mammal.
Not to be confused with "dog".
------------------------------
Enter a word to lookup:
'whale' is not in this dictionary!
Enter a word to lookup:
---------------------
DOG:
A four legged mammal.
---------------------
Enter a word to lookup:
------------------------------
LIZARD:
A four legged scalled reptile.
------------------------------
Enter a word to lookup:
